3.evaluate_model/scripts/nbconverted/get_LOIO_probabilities.py

```diff
@@ -205,7 +192,7 @@
 
 # define save path
 compiled_LOIO_save_path = pathlib.Path(
-    f"{LOIO_probas_dir}/compiled_LOIO_probabilites_withshuffled.tsv"
+    f"{LOIO_probas_dir}/compiled_LOIO_probabilites_withshuffled_withnoic.tsv"
 )
```

Comment:
Is there a reason to not save this file as `compiled_LOIO_probabilites.tsv` and delete the rest of the files? This file should include everything (shuffled, no ic) so I think to avoid confusion it would be best to just have this one file in the repo.

It would also help maintain consistency with the other evaluate results. For example, `F1_scores/` has one file (`compiled_F1_scores.tsv`) with all the parameter combos (data_split	shuffled	feature_type	balance_type	dataset_type).